README.md

```diff
@@ -30,6 +30,23 @@ The gradebook builds on the awesome work of @jwo and @matthiasak. Each student y
 
 Data is cached in localStorage so as not to DDoS TIYO (too much). Clicking the "Refresh Grades" button will scrape the site once again and store the new data.
 
+### Homework
+
+The Homework module builds upon the inspiration from @jah2488.  It allows communication to an authorized ruby process running on your computer to rapidly clone and setup git based projects on your computer.  This allows for easy cloning of PR's, Git Repos, Gists, and various other git related links.  For full documentation please read more about the underlying tool newline_hw [here](https://github.com/TIYDC/newline_hw#newlinehw).
+
```

Comment:
Can we add a bit on how to use it? Basically: install deps, go to assignment/submission page, click link... magic. And add in the animated GIF maybe?